assets/app/components/alertBanner.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 
 const propTypes = {
-  message: React.PropTypes.string.isRequired
+  message: React.PropTypes.string.isRequired,
+  type: React.PropTypes.string.isRequired
 };
 
 const AlertBanner = (props) =>
   !props.message ? null :
   <div className="usa-grid">
     <div className="alert-container container">
-      <div className="usa-alert usa-alert-error new-site-error" role="alert">
+      <div
+        className={`usa-alert usa-alert-${props.status} new-site-error`}
```

Comment:
Maybe follow-up on what this class is doing with @maya? This might have been changed....